components/table/demo/paging.md

```diff
@@ -11,7 +11,7 @@ title:
 
 ## en-US
 
-when use pagination in table.
+when using pagination in table:
```

Comment:
When using pagination in table, and you can use `pagination={false}` to turn off pagination.